collections/misc/collprofiles.php

```diff
@@ -39,7 +39,7 @@
 <html lang="<?php echo $LANG_TAG ?>">
 <head>
 	<title><?php echo $DEFAULT_TITLE . ' ' . ($collid && isset($collData[$collid])? $collData[$collid]['collectionname'] : ''); ?></title>
-	<meta name="keywords" content="Natural history collections,<?php echo ($collid ? $collData[$collid]['collectionname'] : ''); ?>" />
+	<meta name="keywords" content="Natural history collections,<?php echo ($collid && array_key_exists($collid, $collData) ? $collData[$collid]['collectionname'] : ''); ?>" />
```

Comment:
Just a random bug fix if a user uses a coll Id that doesn't have any data in the url. Shouldn't happen organically, but nice to have the graceful erroring anyway.